src/VisualStudio/LiveShare/Impl/FindAllReferencesHandler.cs

```diff
@@ -15,16 +15,18 @@
 using Microsoft.CodeAnalysis.LanguageServer;
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService;
 using Microsoft.VisualStudio.LiveShare.LanguageServices;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Text.Adornments;
+using Newtonsoft.Json.Linq;
 using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.VisualStudio.LanguageServices.LiveShare
 {
     // TODO - This should move to the ILanguageClient when we remove the UI thread dependency.
     // https://github.com/dotnet/roslyn/issues/38477
-    internal class FindAllReferencesHandler : ILspRequestHandler<LSP.ReferenceParams, object[], Solution>
+    internal class FindAllReferencesHandler : ILspRequestHandler<object, object[], Solution>
```

Comment:
why object?

Reply:
Similar reasons to the rest - the input has IProgress<T> which liveshare can't deal with, and the output is sumtype, so object for both.  Will add a comment explaining.